nav2_map_server/README.md

```diff
@@ -63,41 +138,106 @@ free_thresh: 0.196
 ```
 
 The Nav2 software retains the map YAML file format from Nav1, but uses the ROS2 parameter
-mechanism to get the name of the YAML file to use. This effectively introduces a
-level of indirection to get the map yaml filename. For example, for a node named 'map_server',
-the parameter file would look like this:
+mechanism to get the name of the YAML file to use. This effectively introduces a level of indirection to get the map yaml filename. For example, for a node named 'map_server', the parameter file would look like this:
 
-```
-# map_server_params.yaml
+```yaml
+# map_server_params_2d.yaml
 map_server:
     ros__parameters:
-        yaml_filename: "map.yaml"
+        yaml_filename: "map_2d.yaml"
 ```
 
-One can invoke the map service executable directly, passing the params file on the command line,
-like this:
+Below is the process to use `parameters file` with ROS2:
 
+```shell
+# Provide the params filename accordinaly
+# Specify the parmeters file
+############################
+# For 2D Map Server
+$ ros2 run nav2_map_server map_server_2d --ros-args --params-file <map_server_params_2d.yaml>
+# For 3D Map Server
+$ ros2 run nav2_map_server map_server_3d --ros-args --params-file <map_server_params_2d.yaml>
 ```
-$ map_server __params:=map_server_params.yaml
-```
-
-There is also possibility of having multiple map server nodes in a single process, where the parameters file would separate the parameters by node name, like this:
 
+For 3D map server YAML consists of two main arguments file name as `image`, and view point as `origin` :
+```yaml
+image: testmap.pcd
+origin: [0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0]
 ```
-# combined_params.yaml
-map_server1:
+and the `parameters file` can be  defined as shown below:
+ 
+```yaml
+# map_server_params_3d.yaml
+map_server:
     ros__parameters:
-        yaml_filename: "some_map.yaml"
+        yaml_filename: "map_3d.yaml"
+```
```

Comment:
This block is better to be placed before "Below is the process to use parameters file with ROS2: ros2 run nav2_map_server ..." listings. Otherwise, we are out of the sequence of presentation.